src/Observable.ts

```diff
@@ -127,7 +20,7 @@ export type ObservableInput<T> = ObservableOrPromise<T> | ArrayOrIterator<T>;
  *
  * @class Observable<T>
  */
-export class Observable<T> implements CoreOperators<T>  {
+export class Observable<T> {
```

Comment:
They are, I don't see a real need for them, though if we needed too we could bring them back I suppose.  After all the file was, was a list of signatures and to an interface, which is what we've replaced with this PR.